From 3423d1055677feabbddc92ce49068c785fa4187a Mon Sep 17 00:00:00 2001
From: Link Dupont <link@sub-pop.net>
Date: Thu, 2 May 2024 13:01:21 -0400
Subject: [PATCH] fix: Update metainfo

- Use <developer> tag instead of deprecated <developer_name>.
- Shorten summary.

Signed-off-by: Link Dupont <link@sub-pop.net>
---
 data/app.drey.Damask.metainfo.xml.in.in | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/data/app.drey.Damask.metainfo.xml.in.in b/data/app.drey.Damask.metainfo.xml.in.in
index 85eef70..18c9b9d 100644
--- a/data/app.drey.Damask.metainfo.xml.in.in
+++ b/data/app.drey.Damask.metainfo.xml.in.in
@@ -23,8 +23,10 @@
   <launchable type="desktop-id">@APP_ID@.desktop</launchable>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-3.0-or-later</project_license>
-  <developer_name>Link Dupont</developer_name>
-  <summary>Automatically set a background from the Internet</summary>
+  <developer id="@subpop@floss.social">
+    <name>Link Dupont</name>
+  </developer>
+  <summary>Automatic wallpapers from the Internet</summary>
   <description>
     <p>Damask is an automatic wallpaper application. It currently supports setting the wallpaper image from the following sources:</p>
     <ul>
-- 
2.44.0
